### DISCUSSION TITLE
How do I run this thing?

### DISCUSSION THREAD

**Discussion by applecuckoo** [pleonex/PlayMobic#5] [Q&A] [answered]
Hi there! I've just built this on a 64-bit Linux computer with `dotnet script build.csx --isolated-load-context -- --target=Default` and I have no idea how to run it. It might be a bit more helpful to write some run instructions - I installed the .NET SDK just for this!

**Comment by pleonex** [comment, marked as answer]
The project is not finished yet. I'll write some installation and usage instructions before releasing it.

If you want to try a development build, you can compile it as you did. Just run also the target bundle to create zip files with the programs: `dotnet script build.csx --isolated-load-context -- --target=Bundle`. The programs will be in a new folder in `build/artifacts`.
